plugins/cilium-cni/cni-install.sh

```diff
@@ -36,8 +36,7 @@ else
 	cat > ${CNI_CONF_NAME} <<EOF
 {
     "name": "cilium",
-    "type": "cilium-cni",
-    "mtu": ${MTU}
```

Comment:
Is the mtu config a typo or just no longer used?

Reply:
@raybejjani it was removed here https://github.com/cilium/cilium/commit/829fc5730dc1741a07f8781ce6fc74eaeba4edc9#diff-b48f598e63ec7de2b642021d956206ce the trailing "," in the `"type": "cilium-cni",` is the typo

Reply:
@raybejjani The mtu is now configured centrally via the cilium-agent so it applies to all plugins with encap overhead accounting taken into account